apps/docs/src/lib/pages/composables/use-browser-context.md

```diff
@@ -21,6 +21,6 @@ If you are using SvelteKit, then you'll want to use the page store rather than t
 
 ## Definition
 
-```ts
+```js
```

Comment:
There seemed to be a problem in almost every file where if they had this ts code block, it would raise the following error:
```
[vite] Internal server error: Error while preprocessing /Users/Caladan/Code/sveltekit-migration/apps/docs/src/lib/pages/dates/month.md - The language "typescript" has no grammar.
```

Reply:
Hum, I'll check that out when I have time this week. Either way, not really problematic, we can always fix it later.

Reply:
Yeah, either way, the formatting is the same, so it is the same